recipes-extended/xen/xen-tools_git.bbappend

```diff
@@ -9,4 +9,6 @@ require xen-common.inc
 require ${@bb.utils.contains('DISTRO_FEATURES', 'blktap2', 'xen-tools-blktap2.inc', 'xen-tools-blktap3.inc', d)}
 require xen-tools-openxt.inc
 
+B = "${S}"
```

Comment:
May be worth a comment saying "workaround for setuptools3 overriding autotools-brokensep"